### PR TITLE
Save client max daily usage

### PR DESCRIPTION
**Pull request by Kaian**
Persist client max daily on database periodically

#### Type Of Change 
- [ ] Small bug fix
- [X] New feature or enhancement
- [ ] Breaking change

#### Checklist:
- [X] Commits are named and have tag following [commit rules](https://github.com/irontec/ivozprovider/blob/bleeding/doc/dev/en/commits.md)
- [X] Commits are split per component (schema, web/admin, kamusers, agis, ..)
- [X] Changes have been tested locally
- [ ] Fixes an existing issue (Fixes #XXXX)

### CODE REVIEW

File: library/Ivoz/Provider/Infrastructure/Persistence/Doctrine/Mapping/Company.CompanyAbstract.orm.yml [hunk 1]
```diff
@@ -59,6 +59,21 @@ Ivoz\Provider\Domain\Model\Company\CompanyAbstract:
...
         default: 1000000
       column: maxDailyUsage
+    currentDayUsage:
+      type: decimal
+      nullable: true
+      precision: 10
+      scale: 4
+      options:
+        default: 0
+      column: currentDailyUsage
```
Comment (Kaian) [inline library/Ivoz/Provider/Infrastructure/Persistence/Doctrine/Mapping/Company.CompanyAbstract.orm.yml:69]: currentDayUsage for column name

File: library/Ivoz/Provider/Domain/Service/Company/SyncDailyUsage.php [hunk 2]
```diff
@@ -0,0 +1,86 @@
...
+        $this->client = $client;
+        $this->companyRepository = $companyRepository;
+    }
+
+    /**
+     * @return void
+     */
+    public function updateAll()
+    {
+        $this->logger->info('Companies balances are about to be synced');
```
Comment (Kaian) [inline library/Ivoz/Provider/Domain/Service/Company/SyncDailyUsage.php:35]: Company daily usage :)

File: library/Ivoz/Provider/Domain/Service/Company/SyncDailyUsage.php [hunk 3]
```diff
@@ -0,0 +1,86 @@
...
+{
+    protected $entityTools;
+    protected $logger;
+    protected $client;
+    protected $companyRepository;
+
+    public function __construct(
+        EntityTools $entityTools,
+        LoggerInterface $logger,
+        CompanyBalanceServiceInterface $client,
```
Comment (Kaian) [inline library/Ivoz/Provider/Domain/Service/Company/SyncDailyUsage.php:21]: CompanyBalanceServiceInterface ? :thinking:

File: library/Ivoz/Provider/Domain/Service/Company/SyncDailyUsage.php (hunk 2 above)
Comment (mmadariaga) [reply library/Ivoz/Provider/Domain/Service/Company/SyncDailyUsage.php]: Thanks :)

**Review by cruzccl — APPROVED** [review]

**Pull request merged**